vnext/Desktop/packages.config

```diff
@@ -7,7 +7,7 @@
   <package id="Microsoft.SourceLink.Common" version="1.0.0" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.SourceLink.GitHub" version="1.0.0" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.200615.7" targetFramework="native" />
-  <package id="ReactNative.Hermes.Windows" version="0.1.6" targetFramework="native" />
+  <!-- package id="ReactNative.Hermes.Windows" version="0.5.0-f56606a8" targetFramework="native" / -->
```

Comment:
remove?

Reply:
We want to make it easy for people to try Hermes, by just uncommenting a few lines of code and setting something to true.

---
In reply to: [467275649](https://github.com/microsoft/react-native-windows/pull/5666#discussion_r467275649) [](ancestors = 467275649)